src/main/resources/view/MainWindow.fxml

```diff
@@ -32,28 +38,26 @@
             <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
           </Menu>
         </MenuBar>
-
-        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
+        <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
           <padding>
-            <Insets top="5" right="10" bottom="5" left="10" />
+            <Insets top="10" right="10" bottom="10" left="10" />
           </padding>
-        </StackPane>
-
+          <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS" style="-fx-background-color: white;"/>
+        </VBox>
         <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="pane-with-border"
-                   minHeight="100" prefHeight="100" maxHeight="100">
+                   minHeight="100" prefHeight="100" maxHeight="100" style="-fx-background-color: #4d4e4f;
+                   -fx-border-color: #4d4e4f;">
```

Comment:
Great job of adhering to the checkstyles by reformatting the codebase to add 8 spacing for the next line!